cmd/utils/customflags.go

```diff
@@ -46,7 +44,6 @@ func (self *DirectoryString) Set(value string) error {
 // Custom cli.Flag type which expand the received string to an absolute path.
 // e.g. ~/.ethereum -> /home/username/.ethereum
 type DirectoryFlag struct {
-	cli.GenericFlag
```

Comment:
If pending since Dec 11, that's one month + 1 day. According to your logic, it's already a "couple" ;)